docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,366 @@
+
+Package implicitly relies on marathon as the definition provided by package is converted to a marathon app template. However, marathon doesn't know about package. By the end of this guide, you will be able to build, publish, and browse your package in the cluster.
+
+
+### This repository
+- Current packaging version is v4 and we follow this standard for this guide. This guide will be updated as and when we release a new version.
+- The packages are located in `/repo/packages` folder.
+- This tutorial is in `/tutorial` folder
+- You can refer to to **schema** in `/repo/meta/schema` folder. This folder has
+  - config-schema.json that refers to the schema of the config.json
+  - package-schema.json that refers to the schema of the package.json
+  - v3-resource-schema.json that refers to the schema of the resource.json for the v3 and v4 packages
+  - repo-schema.json files are not meant to be used by a developer.
+
+## Create a package
+Let us build a simple python http server, which, when receives a Get request, responds with the current time at the server. We will start with this and build a package that provides this python server as a service.
+
+
+### Step 1 : Create a simple python http server
+For the purposes of this guide, we will be using python3 and the BaseHTTPServer library it has. Let's create a file called `helloworld.py` in an empty directory called `time-server`
+
+```python
+import time
+import BaseHTTPServer
+import os
+
+
+HOST_NAME = '0.0.0.0' # Host name of the http server
+# Gets the port number from $PORT0 environment variable
+PORT_NUMBER = int(os.environ['PORT0'])
+
+class MyHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+    def do_GET(s):
+        """Respond to a GET request."""
+        s.send_response(200)
+        s.send_header("Content-type", "text/html")
+        s.end_headers()
+        s.wfile.write("<html><head><title>Time Server</title></head>")
+        s.wfile.write("<body><p>The current time is %s</p>" % time.asctime())
+        s.wfile.write("</body></html>")
+
+
+if __name__ == '__main__':
+    server_class = BaseHTTPServer.HTTPServer
+    httpd = server_class((HOST_NAME, PORT_NUMBER), MyHandler)
+    print time.asctime(), "Server Starts - %s:%s" % (HOST_NAME, PORT_NUMBER)
+    try:
+        httpd.serve_forever()
+    except KeyboardInterrupt:
+        pass
+    httpd.server_close()
+    print time.asctime(), "Server Stops - %s:%s" % (HOST_NAME, PORT_NUMBER)
+
+```
+
+The code snippet simply starts a python server and serves the get or post requests with a html text that says the current time. You should be able to run this code snippet with `python helloworld.py` and browse [localhost:8000](http://localhost:8000).
```

Comment:
Changed the code to read the port statically and then added a subsection on how to use with environment varialble.